icekit/project/urls.py

```diff
@@ -40,8 +40,11 @@
         {'sitemaps': sitemaps}),
 
     # Installed apps.
-    url(r'^api/pages/', include('icekit.pages_api.urls')),
     url(r'^forms/', include('forms_builder.forms.urls')),
+    # APIs
+    # TODO This URL routing technique doesn't show all endpoints in root /api/
+    url(r'^api/', include('icekit.pages_api.urls')),
```

Comment:
Worth addressing now I think. I'd suggest doing the router registration here, not in each separate `urls.py`